tsdb/mapper.go

```diff
@@ -184,7 +184,7 @@ func (rm *RawMapper) NextChunk() (interface{}, error) {
 
 // Close closes the mapper.
 func (rm *RawMapper) Close() {
-	if rm.tx != nil {
+	if rm != nil && rm.tx != nil {
```

Comment:
Not strictly part of this change, but nil mappers are possible, so this protects against someone calling `Close` on one.